### PR TITLE
Ruby 4.5 Solo Challenge

### PR DESCRIPTION
**Pull request by tobylad**
Tagging @Bontgoy for code review!

### CODE REVIEW

File: ruby/vampires.rb [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+  end
+
+
+  puts "How old are you?" 
+  age = gets.to_i
+
+  puts "What year were you born?"
+  birthyear = gets.to_i
+
+  if age == (2017 - birthyear)
```
Comment (Bontgoy) [inline ruby/vampires.rb:27]: I would like to see you follow the age convention in the instructions.  We want to know if you the employee is lying about their age.

Lets consider checking the current year against how old they say they are and what year they say that they were born in.  Investigate the Ruby `Time` class and see if you can come up with something creative and dynamic.

File: ruby/vampires.rb [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  allergy = "" 
+    puts "Do you have any allergies?  Type done when finished."
+    until allergy == "done" || allergy == "sunshine"
+      puts "Name an allergy."
+      allergy = gets.chomp     
+    end 
+
+  if allergy == "sunshine"
+    allergy = true
+  else allergy = false
```
Comment (Bontgoy) [inline ruby/vampires.rb:66]: Make sure that you are nesting the actions within `else` and not next to it.

e.g
```
if allergy == "sunshine"
  allergy = true
else
  allergy = false
end
```

File: ruby/vampires.rb [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+# assumed that the new employee is a vampire.  If the employee has
+# a sunshine allergy, they are probably a vampire.
+
+  d_vampire = false
+
+  if employee_name == false 
+    then d_vampire = true
+  end 
+
+  unless res_inc == true
```
Comment (Bontgoy) [inline ruby/vampires.rb:81]: This is wild but you totally made it work ;)

100 points for determination here but keep in mind for the future that if you start nesting conditions this deeply there may be a simpler way to track and organize your data.

**Review by Bontgoy — COMMENTED** [review]
@majesty86 Good work here!  I left a few design comments but I like what I see.

:shipit:

File: ruby/vampires.rb (hunk 3 above)
Comment (tobylad) [reply ruby/vampires.rb:81]: @Bontgoy Thanks for the review! I'd definitely like to come back to this after further learning and clean it up.

**Pull request merged**